content/docs/shared-content/lr-notice.md

```diff
@@ -0,0 +1,3 @@
+<Admonition type="important" title="Enrollment Pause for Logical Replication Beta">
+We have temporarily paused new enrollments in our Logical Replication Beta to evaluate feature performance and incorporate feedback from our early adopters. Please stay tuned for updates, and thank you for your interest. We plan to reopen enrollment again soon.
```

Comment:
```suggestion
We have temporarily paused new enrollments in our Logical Replication Beta program. This pause is aimed at evaluating the feature's performance and incorporating feedback from our early adopters. Please stay tuned for updates, and thank you for your interest. We plan to reopen enrollment again soon.
```

Reply:
Nitpick, I know. I just found that I had to read the first sentence again to grok it. LGTM otherwise!